frontend/src/styles/training-page.scss

```diff
@@ -1,79 +1,67 @@
-.register {
-	margin-top: 5vh;
+.trainingPageContainer {
+	margin-top: 2vh;
 
-	.registerContainer {
+	.alertContainer {
 		display: flex;
-		align-items: center;
+		justify-content: center;
+		margin-bottom: 4rem;
 
-		.imgContainer {
-			img {
-				display: none;
-
-				@media screen and (min-width: 768px) {
-					display: block;
-				}
-			}
-		}
-	}
-
-	.registerContent {
-		width: 400px;
-		min-height: 550px;
-		display: flex;
-		flex-direction: column;
-		border-radius: 19px;
-		margin: 5vh auto;
-		// border: 2px solid #f6921e;
-		background: #faf3ed;
-		padding: 2rem;
-		box-shadow: 10px 10px 8px $secondary;
-
-		.formH1 {
+		.alert {
+			z-index: 1;
+			width: 60%;
 			text-align: center;
-			margin-bottom: 1rem;
-		}
-
-		.registerForm {
-			display: flex;
-			flex-direction: column;
+			align-self: center;
+			@media screen and (max-width: 1500px) {
```

Comment:
max-width says target from the specified screen width and below.

min-width says target from the specified screen width and above.

For a mobile first approach, min width of >=800px may be used.